rules/validator/validator.go

```diff
@@ -121,12 +121,29 @@ func (v *validator) validateMappingRules(mrv []view.MappingRule) error {
 
 		// Validate the aggregation ID.
 		if err := v.validateAggregationID(rule.AggregationID, firstLevelAggregationType, types); err != nil {
-			return fmt.Errorf("mapping rule '%s' has invalid aggregation id %v: %v", rule.Name, rule.AggregationID, err)
+			return fmt.Errorf("mapping rule '%s' has invalid aggregation ID %v: %v", rule.Name, rule.AggregationID, err)
 		}
 
-		// Validate that the storage policies are valid.
-		if err := v.validateStoragePolicies(rule.StoragePolicies, types); err != nil {
-			return fmt.Errorf("mapping rule '%s' has invalid storage policies in %v: %v", rule.Name, rule.StoragePolicies, err)
+		// Validate the drop policy is valid.
+		if !rule.DropPolicy.IsValid() {
+			return fmt.Errorf("mapping rule '%s' has an invalid drop policy: value=%d, valid_values=%v",
+				rule.Name, int(rule.DropPolicy), policy.ValidDropPolicies())
+		}
+
+		// Validate the storage policies if drop policy not active, otherwise ensure none.
+		if rule.DropPolicy.IsDefault() {
```

Comment:
I realize this is not within the scope of this diff, but would `NotActive` have been a better name for this?

Reply:
It seemed more consistent at the time to use IsDefault considering all the other IsDefault checks for StoragePolicies, AggregationID, etc.  We can assess this later, it's thankfully a very straight forward change if we do want to do this later.